native/src/modules/layout/__tests__/HeaderContainer.spec.js

```diff
@@ -56,74 +114,47 @@ describe('HeaderContainer', () => {
         },
         switchingLanguage: false
       },
-      contentLanguage: 'de',
+      contentLanguage: 'en',
       cities: {
         status: 'ready',
         models: [city]
       }
     }
   }
 
-  it('shareUrl should be set to path for categories route', () => {
-    jest.doMock('../components/Header', () => Header)
+  const render = props => {
     const HeaderContainer = require('../containers/HeaderContainer').default
-
-    const categoriesRouteMapping = {
-      routeKey1: {
-        status: 'ready',
-        path: 'abc',
-        depth: 1,
-        language: language._code,
-        city: city.name,
-        allAvailableLanguages: new Map(),
-        models: {},
-        children: {}
-      }
-    }
-
-    const state: StateType = prepareState({}, categoriesRouteMapping, {})
-    const store = mockStore(state)
-
-    const ownProps = {
-      scene: {
-        route: {
-          key: 'routeKey1'
-        }
-      }
-    }
-
-    const result = TestRenderer.create(
+    return TestRenderer.create(
       <Provider store={store}>
         {/* $FlowFixMe not all props passed */}
-        <HeaderContainer {...ownProps}/>
+        <HeaderContainer {...props}/>
       </Provider>
     )
+  }
 
+  const assert = (result, expected) => {
```

Comment:
```suggestion
  const assertProps = (result, expected) => {
```
Also not sure if `result` is a good param name.

Reply:
done